bokeh/models/tests/test_annotations.py

```diff
@@ -122,6 +135,43 @@ def test_BoxAnnotation():
         "level",
     ], LINE, FILL)
 
+def test_Label():
+    label = Label()
+    assert label.plot is None
+    assert label.level == 'annotation'
+    assert label.x is None
+    assert label.y is None
+    assert label.x_units == 'data'
+    assert label.y_units == 'data'
+    assert label.text ==  'text'
+    assert label.angle == 0
+    assert label.x_offset == 0
+    assert label.y_offset == 0
+    assert label.render_mode == 'canvas'
+    assert label.x_range_name == 'default'
+    assert label.y_range_name == 'default'
+    assert isinstance(label.source, ColumnDataSource)
+    assert label.source.data == {}
+    yield check_text, label
+    yield check_background, label, None, 1.0
+    yield check_border, label, None, 1.0, 0.0
```

Comment:
There's an actual unit test failure: https://travis-ci.org/bokeh/bokeh/jobs/121332793#L2231-L2231

I think the `0.0` here at the end is the problem